Dockerfile

```diff
@@ -73,7 +73,6 @@ RUN go build -o bin/minioServer src/backend/minioServer.go
 RUN go build -o bin/users src/credential/users.go
 RUN go build -o bin/projects src/credential/projects.go
 RUN go build -o bin/master src/frontend/master.go
-RUN src/tools/generate_certificates
```

Comment:
I made a mistake by proposing to remove it from the Docker environment. We must leave it there. The issue is that the certificates generated will be in /app within the Docker image (which is ok in the end).
We need to update the start_container script
and edit 
export TLS_CERT_PATH=/app/src/tools/certificat.crt
to point to
export TLS_CERT_PATH=/app/certificat.crt
And add to Dockerfile
RUN scripts/generate_certificates

Like that the generate_certificates is general purpose and is executed within the container environment